src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submission_details.cljs

```diff
@@ -126,9 +126,43 @@
                    (:workflows props))}])))})
 
 
+(react/defc AbortButton
+  {:kill-submission
+   (fn [{:keys [props state this]}]
```

Comment:
In discussions with David Mohs, we are of the opinion that we should be avoiding having state passed into child react components. Instead, they should be passed only what they need as arguments and state-handling call-backs.